localops/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "localops"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+
+[target.'cfg(unix)'.dependencies]
+nix = { version = "0.25", features = ["signal", "process"]}
+
+[target.'cfg(macos)'.dependencies]
+nix = { version = "0.25", features = ["signal", "process"]}
```

Comment:
am I holding this right? I have no idea what is even going on

Reply:
`unix` includes `macos` I believe, there is `linux` which does not